src/olmo_core/nn/attention/__init__.py

```diff
@@ -55,21 +55,60 @@
 
 @dataclass
 class SlidingWindowAttentionConfig(Config):
-    pattern: List[bool]
-    window_size: int = 4096
-    force_first: bool = True
-    force_last: bool = True
+    pattern: List[int]
+    """
+    The pattern of window sizes to use for attention, repeated to cover all layers.
+    A value of -1 indicates full attention. For example, a pattern of `[4096, 4096, 4096, -1]`
```

Comment:
Our docs builder parses doc strings as reStructuredText (RST), which requires double backticks for inline code.

```suggestion
    A value of -1 indicates full attention. For example, a pattern of ``[4096, 4096, 4096, -1]``
```